cli.js

```diff
@@ -73,8 +73,25 @@ function filterProcesses(input, processes, flags) {
 		});
 }
 
+function handleFkillError(processes) {
+	const suffix = processes.length > 1 ? 'es' : '';
+	if (process.stdout.isTTY === false) {
+		console.log(`Error killing process${suffix}. Try \`fkill --force ${processes.join(' ')}\``);
+	} else {
+		return inquirer.prompt([{
+			type: 'confirm',
+			name: 'forceKill',
+			message: `Error killing process${suffix}. Would you like to use the force?`
```

Comment:
You're right - though are we not contemplating adding 'kill all' functionality at a later stage (on other issues) - this would future proof the messaging no? Happy to remove it though. AFK for the rest of today though so will be a little later.